test/src/test/java/org/corfudb/integration/AbstractIT.java

```diff
@@ -192,6 +193,17 @@ public static void waitForLayoutChange(Predicate<Layout> verifier, CorfuRuntime
         assertThat(verifier.test(refreshedLayout)).isTrue();
     }
 
+    /**
+     * Wait for the Supplier (some condition) to return true.
+     *
+     * @param supplier Supplier to test condition
+     */
+    public static void waitFor(Supplier<Boolean> supplier) {
+        while (!supplier.get()) {
+            Sleep.sleepUninterruptibly(PARAMETERS.TIMEOUT_VERY_SHORT);
```

Comment:
Any reason why `waitForEpochChange` sleeps every  `TIMEOUT_SHORT`, but this uses `TIMEOUT_VERY_SHORT`?

Reply:
Not really. We can use TIMEOUT_VERY_SHORT for all the waits for faster layout discovery and test completion.

Reply:
Ok.